src/encoder.c

```diff
@@ -49,39 +53,55 @@ int32_t get_encoder_right_diff_count(void)
 }
 
 /**
+ * @brief Read left motor encoder total count.
+ *
+ * The total count is simply the sum of all encoder counter differences.
+ */
+int32_t get_encoder_left_total_count(void)
+{
+	return left_total_count;
+}
+
+/**
+ * @brief Read right motor encoder total count.
+ *
+ * The total count is simply the sum of all encoder counter differences.
+ */
+int32_t get_encoder_right_total_count(void)
+{
+	return right_total_count;
+}
+
+/**
  * @brief Read left motor encoder travelled distance in millimeters.
  */
-float get_encoder_left_distance(void)
+int32_t get_encoder_left_micrometers(void)
 {
-	return left_distance;
+	return left_micrometers;
 }
 
 /**
  * @brief Read right motor encoder travelled distance in millimeters.
```

Comment:
Change the header to micrometers or the variable to millimeters (also for left motor encoder)

Reply:
Good catch! :blush: